server.js

```diff
@@ -2,6 +2,11 @@ const express = require('express');
 const cors = require('cors');
 const connectDB = require('./config/db');
 const path = require('path');
+const res = require('express/lib/response');
```

Comment:
Maybe pull out the nodemailer from the server file and keep it separate module?

Reply:
I agree. Probably keep it separate